Pod/Classes/AGWindowsBoundsSizes.h

```diff
@@ -0,0 +1,76 @@
+//
+//  AGWindowsBoundsSizes.h
+//  AGSnapshotHelper
+//
+//  Created by Adam Grzegorowski on 14/04/16.
+//  Copyright © 2016 Allegro Group. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#pragma mark - iPhone
+
+/**
+ Struct of iPhone possible application window bounds.
+ */
+struct AGiPhoneWindowBoundsSizes {
+    CGSize portrait;
+    CGSize landscape;
+};
+
+typedef struct AGiPhoneWindowBoundsSizes AGiPhoneWindowBoundsSizes;
+
+#pragma mark - iPad
+
+/**
+ Struct of iPad possible application window bounds.
+ */
+struct AGiPadWindowBoundsSizes {
+
+    struct fullscreen {
```

Comment:
I think `fullscreen` is correct
![screen shot 2016-06-06 at 11 58 21](https://cloud.githubusercontent.com/assets/9446070/15818536/42512726-2bde-11e6-8013-5a193b943a1a.png)